setup.py

```diff
@@ -62,14 +60,12 @@
     license='MIT license',
     zip_safe=False,
     keywords='azure iot edge dev tool',
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <3.8',
+    python_requires='>=3.6, <3.8',
     classifiers=[
         'Development Status :: 2 - Pre-Alpha',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
         'Natural Language :: English',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7'
```

Comment:
Add the Python 3.8 metadata classifier if 3.8 is supposed to be supported.

Reply:
Sorry, that was a typo. Will add this in in a separate PR when it has been tested.